.github/workflows/on-tag-push-create-release.yml

```
@@ -0,0 +1,269 @@
+jobs:
+  common-vars:
+    name: Setup Release Variables
+    runs-on: ubuntu-latest
+    outputs:
+      tag-name: ${{ github.ref_name }}
+      release-name: ${{ steps.set-release-name.outputs.release-name }}
+      is-prerelease: ${{ steps.check-prerelease.outputs.is-prerelease }}
+      latest-run-id: ${{ steps.get-latest-build.outputs.run-id }}
+
+    steps:
+    - name: Echo context data
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)